app/src/main/java/com/activity_sync/renderers/base/AdapteeCollection.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2014 Pedro Vicente Gómez Sánchez.
```

Comment:
Yeah, I have written on Trello that I will use existing library with small corrections (adjusted to our code). What should I do in this situation?